scapy/contrib/automotive/obd/pid/pids_00_1F.py

```diff
@@ -7,6 +7,12 @@
 from scapy.fields import StrFixedLenField, BitEnumField, BitField, \
     ScalingField, ConditionalField, FlagsField, XByteEnumField, ShortField
 from scapy.packet import Packet
+import scapy.modules.six as six
+
+if six.PY2:
+    _temperature = "\xC2\xB0C"
+else:
+    _temperature = "\xB0C"
```

Comment:
```
>>> '\xB0C'
'°C'
```

It works with python3.

But this does not:
```
>>> b'\xB0C'
b'\xb0C'
```

What is your suggestion?

Reply:
You are right, your change is actually quite smart

Reply:
Thanks. Do you see a better place for this code? Since I repeat it on all the PID files.

Reply:
The python2 version does not work for Windows.
```python
_temperature = u'\N{DEGREE SIGN}C'
```
Works for me with python 2&3 on Windows and Linux

Reply:
Well, that doesnt work on python2 Linux for me

Reply:
```bash
~ $ python2
Python 2.7.15 (default, Jun 27 2018, 13:05:28) 
[GCC 8.1.1 20180531] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> _temperature = u'\N{DEGREE SIGN}C'
>>> print(_temperature)
°C
```
Confusing. Works fine for me

Reply:
I double checked it. You have to test the `show` function in scapy. On my system, this function produces a different output than simple `print(u'\N{DEGREE SIGN}C')` tests. I add another case for windows in the current implementation.

Reply:
@akorb Could you please double check, that the current implementation works on windows.

Reply:
The current implementation works. I also checked your former implementation with `show` instead of `print` and this works too. Thus this actually already worked for windows with `show`.
```python
if six.PY2:
    _temperature = "\xC2\xB0C"
else:
    _temperature = "\xB0C"
```

Sorry about that.

Reply:
For python3 it works. Python2 throws an error:

```
>>> OBD()/OBD_S01()/OBD_PID05(data=83)
Traceback (most recent call last):
  File "<console>", line 1, in <module>
  File "scapy\packet.py", line 435, in __repr__
    repr(self.payload),
  File "scapy\packet.py", line 435, in __repr__
    repr(self.payload),
  File "scapy\packet.py", line 430, in __repr__
    vcol(val))
  File "scapy\themes.py", line 53, in do_style
    val = str(val)
UnicodeEncodeError: 'ascii' codec can't encode character u'\xb0' in position 5: ordinal not in range(128)
```

This might help: https://stackoverflow.com/a/9942822/2050020